src/main/resources/swagger/api-docs.yaml

```diff
@@ -2181,6 +2181,28 @@ paths:
         500:
           description: Internal Error
 
+  /api/workspaces/tags:
+      get:
+        tags:
+         - Workspaces
+        operationId: getTags
+        summary: |
+          Get all tags used in FireCloud (for autocomplete)
+        parameters:
+          - in: query
+            description: Query string
+            name: q
+            required: false
+            type: string
+        responses:
+          200:
+            description: list of all tags used in FireCloud and their associated frequencies
+            type: array
+            items:
+              $ref: '#/definitions/WorkspaceTag'
```

Comment:
syntax here is not quite right. I think this should be:
```
          200:
            description: list of all tags used in FireCloud and their associated frequencies
            schema:
              type: array
              items:
                $ref: '#/definitions/WorkspaceTag'
```
The swagger plugin for IntelliJ finds these errors!